OpenAction/ActionViewController.swift

```diff
@@ -64,9 +64,16 @@ class ActionViewController: UIViewController {
             var responder = self as UIResponder?
             let selectorOpenURL = sel_registerName("openURL:")
             while let current = responder {
-                if current.responds(to: selectorOpenURL) {
-                    current.perform(selectorOpenURL, with: url, afterDelay: 0)
-                    break
+                if #available(iOS 18.0, *) {
+                    if let application = current as? UIApplication {
+                        application.open(url, options: [:], completionHandler: nil)
+                        break
+                    }
+                } else {
+                    if current.responds(to: selectorOpenURL) {
```

Comment:
As of iOS 18 this `responds` check always returns false which is why this functionality broke